.gitignore

```diff
@@ -11,6 +11,7 @@ dist/
 .coverage
 htmlcov-py27/
 htmlcov-py36/
+htmlcov-py37/
```

Comment:
Can we just ignore all future ones as well?
Something like this: `htmlcov-*`? (not sure if this works).

Reply:
Yes, something like htmlcov-*/ should work. I will try it and let you know.

Reply:
Done!